### ISSUE TITLE
Docker images add support for Linux/ARM64

### ISSUE THREAD

**Issue by DejavuMoe** [ArtalkJS/ArtalkGo#67]
I saw that ArtalkGo has pre-compiled binaries for Linux ARM/ARM64, can you add a Docker image support for ARM/ARM64 architecture to the project, thanks.

**Comment by pluveto** [comment]
Good idea. Currently we're doing a monorepo refactor. After that we'll add such support in a future version.
